bin/test-lxd-network

```diff
@@ -257,11 +269,15 @@ lxc start v1-physical
 waitVMAgent v1-physical
 
 # Interface "eth0" should exist in the VM with the correct MTU.
-lxc exec v1-physical -- ip link show eth0
+lxc exec v1-physical -- ls /sys/class/net/eth0
```

Comment:
I am not sure what `ls` is for? The following `grep` should suffice to check the `eth0` NIC is present. But maybe that's informational?